Update README.md (#13)
DevRel-Foundation/.github: profile/README.md

</details>
<details>
     <summary>
    <strong>What is a Charter?</strong>
  </summary><br />
  The purpose of having a charter for the Developer Relations Foundation is to help people understand its mission and scope. The DevRel Foundation Charter is a living document, allowing the community to propose changes and updates as the project evolves.
</details>
<details>
  <summary>
    <strong>Who is the current Steering Committee?</strong>
  </summary><br />

>   There is currently a temporary Steering Committee. We, the Steering Committee participants, are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined.

Author: ```suggestion
 The current Steering Committee participants are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined.
```

<details>
  <summary>
    <strong>What is a Steering Committee?</strong>
  </summary><br />
  The Steering Committee ensures that the project stays on track of its mission and aligns with the strategic goals defined by the community and working group chairs.  
  In the DevRel Foundation, there is currently a temporary group of volunteers (the Volunteer Steering Committee) who have helped build the initial lightweight processes needed to organize and coordinate the community-driven initiatives.
  
</details>
<details>
  <summary>
    <strong>Who is the current Steering Committee?</strong>
  </summary><br />

>   There is currently a temporary Volunteer Steering Committee. We, the volunteer Steering Committe participants, are composed of tenured Developer Relations professionals, who initiated this “intent to form foundation” built through the LF’s guidance and principles. Once the   charter and mission are in place, a new leadership will be    established. This process will be via elections at a future date to be determined.

Reviewer: ```suggestion
  There is currently a temporary Volunteer Steering Committee. We, the volunteer Steering Committee participants, are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined.
```

>   While our profession is relatively small, it is big enough that there are many really good people who can do this work. And we hope they want to! We had the initiative and time to do this, and so we feel it’s reasonable for us, just like anyone else, to take the first steps       forward on building something

Reviewer: ```suggestion
  While our profession is relatively small, it is big enough that there are many really good people who can do this work. And we hope they want to! We had the initiative and time to do this, and so we feel it’s reasonable for us, just like anyone else, to take the first steps forward on building something
```
